assets/section-featured-blog.css

```diff
@@ -67,16 +67,6 @@
   width: 100%;
```

Comment:
The spacing is the same, actually. The reason it looks different in your screenshots is because the card style is set to standard and not card. 

So it looks like more space, but in reality it's the same. We might want to consider changing the spacing when the card style is set to "card". I posted this on Ludo's PR as well, since I think this would affect the `view-all-button` class for all sections.